### ISSUE TITLE
Update policy of linux_latest in stable channels

### ISSUE THREAD

**Issue by jbgi** [NixOS/nixpkgs#41100]
Is it allowed to be update `linux_latest` to a newer major version of the kernel on a stable channel?

Currently `linux_latest` in 18.03 point to an un-maintained kernel (4.15). This gives of false sense of security to the users that use it.
I suggest that either `linux_latest` be removed from stable channels or kept up-to-date (to the extent that is possible?).

cc @NeQuissimus

**Comment by dezgeg** [comment]
> Is it allowed to be update linux_latest to a newer major version of the kernel on a stable channel?

Yes, please.

> Currently linux_latest in 18.03 point to an un-maintained kernel (4.15).

4.15 should be dropped.

**Comment by NeQuissimus** [comment]
Makes sense. I would not anticipate any trouble. I am already cherry-picking all the LTS kernels into `stable`. I can just do the same for `latest`.
The reason I am currently not doing it is that if any of the kernel modules break, we may be in for additional trouble. (as we currently are with 4.16 and ZFS, which does not work)

**Comment by dezgeg** [comment]
Linux does not have an official out-of-tree module API, so `linux_latest` breaking on out-of-tree modules isn't something that Nixpkgs (neither stable nor unstable) should be concerned about. People who need `zfs` or `nvidia` or other out-of-tree modules should not be using `linux_latest`.
